dali/test/dali_operator_test.h

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2018, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DALI_TEST_DALI_OPERATOR_TEST_H_
+#define DALI_TEST_DALI_OPERATOR_TEST_H_
+
+#include <dali/pipeline/pipeline.h>
+#include <dali/test/graph_descr.h>
+#include <dali/test/tensor_list_wrapper.h>
+#include <dali/test/argument_key.h>
+#include <gtest/gtest.h>
+#include <vector>
+#include <memory>
+#include <map>
+
+namespace dali {
+namespace testing {
+
+using Arguments = std::map<ArgumentKey, double>;  // TODO(mszolucha) some generalization boost::any?
```

Comment:
Please, use `boost::any` or `dali::Value` or something new, but drop this `double` from here.